### DISCUSSION TITLE
repo hidden in organization's public listing, but assessable with link

### DISCUSSION THREAD

**Discussion by bhuffaker** [community/community#21814] [New to GitHub]
<p>I am considering Jupyter Notebooks for a application I am working on. The goal is to embed small notebooks with solutions to small common problems processing Internet related data. I am considering using GitHub repos to store the Notebooks for it’s community feedback support, but I don’t want to clutter my organization’s repo listings with all these little repos. Is there a way to hide a repo from an organization’s public listing, while still making it publicly assessable with a direct link? Alternatively is there a way to create a sub-organization with it’s own repo listings and place all the notebooks there?</p>

**Comment by seveas** [comment, marked as answer]
<p>Neither of these are possible at the moment. Probably the best course of action would be to create a separate organization.</p>
